content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/aging.rst

```diff
@@ -0,0 +1,100 @@
+======================
+Inventory aging report
+======================
+
+.. |SVLs| replace:: :abbr:`SVLs (stock valuation layers)`
+
+The inventory aging report evaluates all items in stock, providing insights into potentially sunken
+purchase costs and delays in profitability.
+
+Create customized pivot tables to analyze product, operation types, month, or company breakdowns.
+This helps identify products in stock that are at risk of passing their expiration or viability
+dates, or instances of rot/decay for fast-expiring items.
+
+.. note::
+   The :guilabel:`Reporting` menu in *Inventory* is only accessible to users with :doc:`admin access
+   <../../../../general/users/access_rights>`.
+
+To access the inventory aging report, go to :menuselection:`Inventory app --> Reporting -->
+Inventory Aging`.
+
+.. _inventory/warehouses_storage/aging-report:
+
+Navigate the inventory aging report
+===================================
+
+By default, the :guilabel:`Inventory Aging` report displays a pivot table, with the month in
+columns, and product category in rows. The default filters, :guilabel:`Incoming` and :guilabel:`Has
+Remaining Qty`, show only products from receipts, and are currently in stock.
+
+:guilabel:`Remaining Qty` displays the number of on-hand items, and :guilabel:`Remaining Value`
+displays the total cost of purchasing these items.
+
+Clicking the :icon:`fa-plus-square` :guilabel:`(plus)` icon in each column or row reveals options to
+expand the pivot table and show a detailed breakdown of the :guilabel:`Remaining Qty` and
+:guilabel:`Remaining Value` by :guilabel:`Product`, :guilabel:`Product Category`, :guilabel:`Date`,
+or :guilabel:`Company`. Clicking the :icon:`fa-minus-square-o` (:guilabel:`minus`) icon collapses it
+back to its previous state.
+
+.. figure:: aging/inventory-aging.png
+   :align: center
+   :alt: Inventory aging report.
+
+   Inventory aging report, showing each **Product** in rows and each reception **Date** in columns,
+   to better monitor products with fast expiration dates.
+
+   Each row shows the the total on-hand quantity and inventory valuation of items purchased on each
+   day.
```

Comment:
This is formatting not as I expected, if this is the desired formatting you can leave it, however, this change looks better to me :)
```suggestion
   to better monitor products with fast expiration dates. Each row shows the the total on-hand
   quantity and inventory valuation of items purchased on each day.
```